src/Xamarin.Android.Build.Tasks/Utilities/ApplicationConfigNativeAssemblyGenerator.cs

```diff
@@ -90,6 +92,9 @@ protected override void WriteSymbols (StreamWriter output)
 				WriteCommentLine (output, "number_of_assemblies_in_apk");
 				size += WriteData (output, NumberOfAssembliesInApk);
 
+				WriteCommentLine (output, "bundled_assembly_name_width");
+				size += WriteData (output, BundledAssemblyNameWidth);
```

Comment:
e.g. perhaps *this* should contain `+1 /* NUL */`?

Reply:
No, as said above, this class merely generates the native assembly - value calculation is not its concern.